File: src/BreakfastProvider.Api/Models/Responses/DailySpecialOrderResponse.cs
namespace BreakfastProvider.Api.Models.Responses;

public record DailySpecialOrderResponse
{
    public Guid OrderConfirmationId { get; init; }
    public Guid SpecialId { get; init; }
    public int QuantityOrdered { get; init; }
    public int RemainingQuantity { get; init; }
}

File: java/breakfast-provider-app/src/main/java/io/lemonlion/breakfast/model/response/DailySpecialOrderResponse.java
package io.lemonlion.breakfast.model.response;

import java.util.UUID;

/** Twin of C# {@code DailySpecialOrderResponse}. */
public record DailySpecialOrderResponse(
        UUID orderConfirmationId, UUID specialId, int quantityOrdered, int remainingQuantity) {
}
